.github/workflows/check-patches.yml

```diff
@@ -0,0 +1,42 @@
+# The action fetches code from the PR's source branch, takes the target branch, and
+# tries to fetch and apply patches using PR's target branch.
+# Example: if a PR targets a version branch `3.20`, the action would checkout to `3.20`
+# branch in the patches repo and try to apply patches from that branch. If patches
+# apply, the action succeeds, otherwise it returns an error, and indicates that patches
+# need to be adjusted.
+#
+# - Action runs on all branches.
+# - Action runs only when label is applied: `check patches` (will be changed to all PRs
+# when workflow is more tested).
+
+name: Test Patches
+
+on:
+  pull_request:
+    types: [labeled, opened, synchronize, unlabeled]
```

Comment:
`unlabeled` seems unnecessary?

Reply:
Original version I wrote used a label to skip the check, so it's a leftover from that solution. Now it's not needed indeed.